thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2813,12 +2813,12 @@ void CThorStats::tallyValue(unsigned __int64 thiscount, unsigned n)
     if (thiscount > max)
     {
         max = thiscount;
-        maxNode = n;
+        maxNode = n+1;
```

Comment:
probably slightly cleaner to pass n+1 into tally() instead.  Especially if this logic migrates to jstats.